stac_geoparquet/arrow/_to_parquet.py

```diff
@@ -19,69 +19,94 @@ def parse_stac_ndjson_to_parquet(
     chunk_size: int = 65536,
     schema: pa.Schema | InferredSchema | None = None,
     limit: int | None = None,
+    schema_version: Literal["1.0.0", "1.1.0"] = "1.0.0",
     **kwargs: Any,
 ) -> None:
     """Convert one or more newline-delimited JSON STAC files to GeoParquet
 
     Args:
         input_path: One or more paths to files with STAC items.
         output_path: A path to the output Parquet file.
+
+    Other args:
```

Comment:
Is this "Other args" from some documentation convention?

Reply:
I used `Other args` for a while to satisfy mkdocstrings, because I had _sworn_ that Keyword Args didn't work, but apparently it is in the list of valid aliases: https://mkdocstrings.github.io/griffe/docstrings/#google-section-other-parameters

So I should switch this to "keyword args"

Reply:
I renamed to Keyword Args 🙏